src/utils/time.ts

```diff
@@ -229,9 +188,10 @@ export function formatDateToWeekday(utcSeconds: number) {
  * @see https://date-fns.org/docs/formatRelative
  */
 export const formatUtcSecondsRelativeToNow = (utcSeconds: number) => {
-  return formatRelative(utcSeconds * ONE_SECOND_IN_MILLISECONDS, Date.now());
+  return formatRelative(fromUnixTime(utcSeconds), Date.now());
 };
```

Comment:
[NB] This can be made a ` liner by removing `{`, `}`, `return`

```typescript
export const formatUtcSecondsRelativeToNow = (utcSeconds: number) => formatRelative(fromUnixTime(utcSeconds), Date.now());
```

Reply:
done: https://github.com/sparkletown/sparkle/commit/61c8d31c7eecfeae9b1debbdc10b22f31a630afe